Blahgger/Blahgger/Data/BlahggerData.cs

```diff
@@ -0,0 +1,137 @@
+﻿using Blahgger.Models;
+using System.Data.SqlClient;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Web.Security;
+
+namespace Blahgger.Data
+{
+    /// <summary>
+    /// This class contains methods for binding data from the database to models.
+    /// </summary>
+    public class BlahggerData
+    {
+        private static BlahggerData singleton = null;
+
+        private BlahggerData() { }
+
+        public static BlahggerData GetInstance()
+        {
+            if (singleton == null)
+            {
+                singleton = new BlahggerData();
+            }
+            return singleton;
+        }
+        
+        public void AddUser(User user)
+        {
+            string sql = @"
+                INSERT INTO Users (Username, HashedPassword)
+                VALUES (@Username, @HashedPassword)
+            ";
+            // TODO Password hashing
+
+            DatabaseHelper.Insert(sql,
+                new SqlParameter("@Username", user.Username),
+                new SqlParameter("@HashedPassword", user.HashedPassword));
+
+        }
+
+        internal int GetUsersId(string username)
```

Comment:
Is the use of `public` and `internal` access modifiers intentional in this class?